$OpenBSD: patch-ports-config_in,v 1.1.1.1 2004/05/09 10:30:22 dons Exp $

Have the old-style config mechanism report the same information as the
package system.

--- ports-config.in.orig	2001-06-24 23:48:35.000000000 +1000
+++ ports-config.in	2004-05-09 15:12:53.000000000 +1000
@@ -73,10 +73,10 @@ while test $# -gt 0; do
   shift
 done
 
-libdir=@libdir@/ports-@PORTS_VERSION@
+libdir=@libdir@/hs-ports-@PORTS_VERSION@
 bindir=@bindir@
 
-syslibs="-package concurrent -package lang -package posix"
+syslibs="-package concurrent -package lang -package posix -package haskell98"
 
 if test "$echo_prefix" = "yes"; then
 	echo $prefix
